Tests/MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -630,7 +630,7 @@ class NavigationServiceTests: TestCase {
         XCTAssertTrue(delegate.recentMessages.contains("navigationService(_:didArriveAt:)"))
     }
 
-    func testProactiveRerouting() {
+    func disabled_testProactiveRerouting() {
```

Comment:
Revival of several disabled tests will be tackled as a tail-work in this ticket: https://github.com/mapbox/mapbox-navigation-ios/issues/3375.